src/store/actions/drinkCategories.js

```diff
@@ -8,15 +8,15 @@ const setFetchedTypes = (actionVal, payload) => ({
 
 export const fetchDrinkTypes = () => (dispatch) => {
   axios
-    .get('https://www.thecocktaildb.com/api/json/v1/1/list.php?c=list')
+    .get('list.php?c=list')
     .then((response) => {
       dispatch(setFetchedTypes(actions.GET_DRINK_TYPES, response.data.drinks));
     });
 };
 
 export const fetchAlcLevel = () => (dispatch) => {
   axios
-    .get('https://www.thecocktaildb.com/api/json/v1/1/list.php?a=list')
+    .get('list.php?a=list')
```

Comment:
move this route to a constant in a separate file, so whenever you need to change it, you will only have to change one line